doc/content/source/executioners/MFEMExecutioner.md

```diff
@@ -0,0 +1,26 @@
+# MFEMExecutioner
+
+## Summary
+
+Virtual base class for executioners used to solve MFEM FE problems.
+
+## Overview
+
+`MFEMExecutioner` is a purely virtual base class used to control the execution of MFEM FE problems.
```

Comment:
Add links to MFEMExecutioner docs.

Reply:
This is the MFEMExecutioner source documentation file - do you mean a self-referential link here?

Reply:
Sorry, wasn't looking closely enough. Ignore this.